svelte/fr/tutorial/17-module-context/01-sharing-code/app-a/AudioPlayer.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+	export let src;
+	export let title;
+	export let composer;
+	export let performer;
+
+	let audio;
+	let paused = true;
+
+	function stopOthers() {
+		// À FAIRE: implémenter stopOther
```

Comment:
```suggestion
		// À FAIRE: implémenter stopOthers
```

c'est peut-être une erreur en anglais mais j'irai pas faire la PR :p